docker/server-jetty/src/main/server-jetty/requirements.txt

```diff
@@ -1,5 +1,6 @@
-adbc-driver-manager==0.1.0
-adbc-driver-postgresql==0.1.0
+adbc-driver-manager==0.2.0
+adbc-driver-postgresql==0.2.0
+connectorx==0.3.1; platform.machine == 'x86_64'
```

Comment:
@devinrsmith should confirm this is the right syntax.

Reply:
He already did and approved images related changes